src/crypto/index.js

```diff
@@ -721,6 +1156,24 @@ Crypto.prototype.saveDeviceList = function(delay) {
 Crypto.prototype.setDeviceVerification = async function(
     userId, deviceId, verified, blocked, known,
 ) {
+    const xsk = this._deviceList.getStoredCrossSigningForUser(userId);
```

Comment:
this could use a comment... this is checking if what's being marked verified is a self or user signing key, and if so, signing it and uploading the sig?